src/object/srv_obj_migrate.c

```diff
@@ -590,9 +593,13 @@ migrate_pool_tls_lookup_create(struct ds_pool *pool, unsigned int version, unsig
 
 		return rc;
 	}
-
-	d_getenv_uint(ENV_MIGRATE_ULT_CNT, &max_migrate_ult);
 	D_ASSERT(generation != (unsigned int)(-1));
+
+	arg.max_ult_cnt = MIGRATE_DEFAULT_MAX_ULT;
+	d_getenv_uint(ENV_MIGRATE_ULT_CNT, &arg.max_ult_cnt);
+	arg.max_mem_size = MIGRATE_DEFAULT_MAX_SIZE;
+	d_getenv_uint64_t(ENV_MIGRATE_MEM_SIZE, &arg.max_mem_size);
```

Comment:
maybe need to with a safe min value to restrict what the ENV can set, because if set it as a very small value possibly cause the rebuild cannot work well. I can change it in one of my following PR so need not refresh specially for this point.

Reply:
need to refresh this PR so let me add this. thanks.